### ISSUE TITLE
Package also CHANGELOG / LICENSE files

### ISSUE THREAD

**Issue by janihur** [intersystems/ipm#671]
Currently only README is included into the compressed source code archive file (`zpm package`).

Include also CHANGELOG / LICENSE files if they are available.

You can of course include all information into README but I have found it very useful to separate the changelog information.

Promote the conventions defined in https://keepachangelog.com/en/1.1.0/ (that's exactly how I do changelogs atm). This also promotes semver.

Considering the open source aspect of the IPM ecosystem the explicit licensing information should be expected from all modules publicly available.

**Comment by isc-tleavitt** [comment]
Agreed on this, good points all around. We also have adopted that changelog convention on all of my team's active open source stuff (including IPM).

**Comment by isc-shuliu** [comment]
Will be fixed in #683
